test/comment/index.js

```diff
@@ -0,0 +1,19 @@
+const bot = require("circle-github-bot").create();
+const fs = require('fs');
+
+const path = '../result/Output.log'
+fs.readFile(path, (err, data) => {
+    if (err) {
+        console.log(`Fail to read file: ${path}, err: ${err.message}`)
+    }
+    const regex = /(?<errors>\d*) Error\(s\), (?<warnings>\d*) Warning\(s\)/gm;
+    const { errors, warnings } = regex.exec(data.toString()).groups
+    if (data.length > 0) {
```

Comment:
`data.length` is always going to be greater than zero even if there are no warnings/errors.
This means you'll always post a comment, no matter what. This may be a thing you want/don't want to do, but make sure you take this into consideration and adjust the code.

Reply:
Good point. I will remove this condition check.